components/dictionary.txt

```diff
@@ -1,11 +1,133 @@
+actin
+ADT
+al
 al.
```

Comment:
I don't think we want both of these, and only "al." is correct, as it is an abbreviation.
```suggestion
al.
```

Reply:
I think this another instance where it doesn't look for punctuation. But this will be obsolete once the old citations from the old notebooks are updated anyway.

Reply:
I'll keep `et` and `al` because spelling won't look at the `.` anyway.